src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -599,7 +599,9 @@ def _serialize_multimedia(self, run_info: Union[FlowRunInfo, NodeRunInfo]):
         # consumed. It is crucial to process the api_calls list in place to avoid losing the reference to the list that
         # holds the generator items, which is essential for tracing generator execution.
         if run_info.api_calls:
-            run_info.api_calls = convert_multimedia_data_to_string(run_info.api_calls, inplace=True)
+            run_info.api_calls = MultimediaProcessor.get_instance().convert_multimedia_data_to_string(
```

Comment:
get_instance not active_instance?

Reply:
get_instance is an extension to active_instance to provide a more optimized error message.
![image](https://github.com/microsoft/promptflow/assets/94442403/19fe479d-34e5-4c8e-8a70-41a2efb90128)